recipes/elasticsearch_configure_xpack.rb

```diff
@@ -0,0 +1,46 @@
+version = node['elasticsearch']['version']
+ca = node['elasticsearch']['ca']
+key_name = node['elasticsearch']['xpack_security_transport_ssl_keystore_path']
+xpack_enabled = node['elasticsearch']['xpack_security_enabled']
+bootstrap_password = node['elasticsearch']['bootstrap_password']
+
+if xpack_enabled && version >= '7.0.0' && version < '8.0.0'
```

Comment:
are we sure es 8 is different?

Reply:
we don't know yet, that's why we don't set a default value for es8.x and user must set them if they choose to use es8.x